plugins/telemetry-otel/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -11,6 +11,7 @@ grant {
     permission java.lang.RuntimePermission "accessDeclaredMembers";
     permission java.net.NetPermission "getProxySelector";
     permission java.net.SocketPermission "*", "connect,resolve";
+    permission java.util.PropertyPermission "*", "read,write";
```

Comment:
```suggestion
    permission java.util.PropertyPermission "*", "read";
```

Reply:
Fair enough, but we need it for 1 single property:

```
permission java.util.PropertyPermission "otel.experimental.sdk.metrics.debug", "read,write";
```

Reply:
Security access check is failing for System.getProperties() call, filter for this particular property comes later.

```
public void checkPropertiesAccess() {
        checkPermission(new PropertyPermission("*",
            SecurityConstants.PROPERTY_RW_ACTION));
    }

```

unfortunately, we need to give full access here.

Reply:
Ah ... I see :( sorry about it, this is bad design (from Otel) for sure